tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -0,0 +1,133 @@
+# path to the private key file for the EDPM nodes
+edpm_privatekey_path: ~/director_standalone/.vagrant/machines/standalone/libvirt/private_key
+# virsh --connect=qemu:///system -q domifaddr director_standalone | awk 'NF>1{print $NF}' | cut -d/ -f1
+edpm_node_ip: 192.168.121.207
```

Comment:
Maybe some of these things should be also added into vars/secrets (where default values don't make much sense or will have to be overriden by practically everyone.

And we should have the text docs for users in addition to the test suite.

That said, i think we can merge this and iterate?

Reply:
actually, I found some errors when I reprovisioned my env (ssh stopped working, I'm working on that)

Reply:
I've added more details, and I think we can merge and iterate now